examples/repository/metadata.staged/snapshot.json

```diff
@@ -1,40 +1,32 @@
 {
  "signatures": [
   {
-   "keyid": "fce9cf1cc86b0945d6a042f334026f31ed8e4ee1510218f198e8d3f191d15309", 
-   "method": "ed25519", 
-   "sig": "f7f03b13e3f4a78a23561419fc0dd741a637e49ee671251be9f8f3fceedfc112e44ee3aaff2278fad9164ab039118d4dc53f22f94900dae9a147aa4d35dcfc0f"
+   "keyid": "182216b8800c50ddf000043b31ddf90d815c754ab4e0b31a5952a839b371bed9",
+   "method": "ed25519",
+   "sig": "01bc8667e2afbceef3df6ec6782088c77e59bdb5a29b75634200528bdedc8af9694e38a672fa2cd00051fb25ac12d3079bbc48f424783d23fb532b7508adf40f"
   }
- ], 
+ ],
  "signed": {
-  "_type": "Snapshot", 
-  "expires": "2030-01-01T00:00:00Z", 
+  "_type": "Snapshot",
+  "expires": "2030-01-01T00:00:00Z",
```

Comment:
Oh, geez. I looked at it backwards and processed it as whitespace added, not removed. Well, the other question stands, I suppose: are the signatures valid?